src/app/sections/hero-banner/hero-banner.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="section parallax-container" id="send-coin-sect">
+  <div class="parallax" materialize="parallax"><img src="./assets/polymorph-background.jpg"></div>
+  <div class="container form-light">
+    <img alt="Navtech logo" id="header-logo" src=".\assets\polymorph-logo.svg" />
+    <div class="row">
+      <div class="col l8 m10 s12 push-l2 push-m1">
+        <div class="center" id="paragraph-container">
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla.</p>
+        </div>
+        <send-coins-form-component lightTheme="true"></send-coins-form-component>
```

Comment:
instead of using boolean here, use a variable called theme and set the theme as a string